src/EventStore.Core/ClusterVNode.cs

```diff
@@ -274,7 +269,10 @@ public class ClusterVNode<TStreamId> :
 
 			var disableInternalTcpTls = options.Application.Insecure;
 			var disableExternalTcpTls = options.Application.Insecure;
-			var enableExternalTcp = ExtTcpOptions.TryParse(out var extTcpOptions);
+			var extTcpOptions = configuration.GetSection("EventStore:Plugins:Tcp").Get<ExtTcpOptions>();
+			var enableExternalTcp = extTcpOptions != null;
```

Comment:
Probably having a explicit config option `EnableExternalTcp` would be nice (same name as the old setting)